packages/react/src/progress-indicator/ProgressIndicatorItem.tsx

```diff
@@ -43,19 +43,19 @@ export interface ProgressIndicatorItemLinkProps extends LinkProps {
 	background?: ProgressIndicatorBackground;
 	/** Determines what style the item displays as.
 	 *
-	 * `blocked` = The user cannot start this step yet.
+	 * `blocked` = This step is not available yet for the user to do.
```

Comment:
I just did some wordsmithing with Grammarly and I think these step descriptions read better:


blocked: This step is not available yet for the user to start.

started: The user has reached this step and begun working on it, but it still needs to be completed.

todo: The step is available for the user to start.

saved: The user has submitted valid data; however, the data still needs to be approved or verified. If no approval is required, use the `done` status instead.

done: The user has submitted valid data and it has been verified (where necessary). This step is complete.

error: There is an error on this step that requires attention.

I think it's worthwhile getting @adrian-yee's opinion before making any changes though

Reply:
blocked: This step is not available yet for the user to start. - agreed
started: The user has reached this step and begun working on it, but it still needs to be completed. – we have removed ‘…is not active or selected’ is this not important?
todo: The step is available for the user to start. - agreed
saved: The user has submitted valid data; however, the data still needs to be approved or verified. If no approval is required, use the done status instead. – ‘The user has submitted valid data; however, the data can be changed by the user or effected by conditional logic prior to this point.?
done: The user has submitted valid data and it has been verified (where necessary). This step is complete. - agreed
error: There is an error on this step that requires attention. – I'm not sure we need this state?

Reply:
@adrian-yee  

> started: The user has reached this step and begun working on it, but it still needs to be completed. - we have removed '...is not active or selected' is this not important?

Yes. We are deprecating the `"doing"` status because it is just the `"started"` status with `isActive: true` applied. So `"started"` now fills the role that the `"doing"` status was filling before.

> ‘The user has submitted valid data; however, the data can be changed by the user or effected by conditional logic prior to this point.?

I would write as this:
"The user has submitted valid data; however, the data can still be changed by the user or affected by conditional logic"

I still think we need a "submitted" status for something that has been submitted for approval but has not been approved yet. We have that status all over the place in BARS.

> The user has submitted valid data and it has been verified (where necessary). This step is complete. 

I'm going to say "approved or verified" instead of just "verified" since the language of "approved" is used a lot in government.

> error: There is an error on this step that requires attention. – I'm not sure we need this state?

This is for when you have saved, moved to the next step but there is invalid data. We have the status now and it would be a breaking change to remove, especially since we have no replacement.

Reply:
@adrian-yee @stowball  I've updated the status description text now